### ISSUE TITLE
Constant function expressions are categorized as variables

### ISSUE THREAD

**Issue by jsejcksn** [denoland/doc_website#114]
Example:

Source:
```ts
/**
 * Log your data
 */
export const log = (data: any): void => console.log(data);
```

Generated:
<img width="168" alt="doc" src="https://user-images.githubusercontent.com/868251/84451824-c8b9c880-ac19-11ea-9eff-e2744e1a6153.png">

It also doesn't seem to recognize types on the anonymous function expression signature.

Is this the intended behavior?

**Comment by lucacasonato** [comment]
This is correct behavior at the moment, because any constant, no matter what you assign it, is a variable. I do get why it would make sense for that to render like a function though. This is actually an issue with denoland/deno upstream though, because that is where the code analysis and documentation generation takes place.

cc @bartlomieju
